queries.sql

```diff
@@ -158,3 +158,65 @@ FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
 
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 1
+WHERE name NOT LIKE '%mon';
+
+UPDATE animals
+SET owner_id = 
+    (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+    WHERE name IN ('Agumon');
+
+UPDATE animals
+SET owner_id = 
+    (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+    WHERE name IN ('Gabumon', 'Pikachu');
+
```

Comment:
> Melody Pond owns Charmander, Squirtle, and Blossom. 
- [ ] You didn't write the code to insert these data above into the DB, and it's definitely going to affect the result of your queries. I'll suggest you add the data and re-run your queries. :+1: :100: